torch/csrc/jit/passes/quantization.cpp

```diff
@@ -1009,7 +1019,9 @@ bool InsertObserversHelper::valueNeedsToBeQuantized(Value* v) {
   }
   // Check whether user is quantizable
   for (const auto& use : v->uses()) {
-    if (nodeQuantizable(use.user) || isDynamicLSTMWeight(v, use, is_dynamic)) {
+    if ((nodeQuantizable(use.user) &&
```

Comment:
&& -> ||?
can you use `valueQuantizable` here?

Reply:
If I make it `||` then I have to remove the op from `_quantizable_call_funcs` else it would insert observers for each input. Removing from `_quantizable_call_funcs` would result in no observers at the output of the node.
I could refactor it a bit more to make the input and output observers for an op explicit and not just depend on `_quantizable_call_funcs`